server/app/tests/models/CarConsumtion.test.js

```diff
@@ -0,0 +1,53 @@
+const assert = require("assert").strict;

```

Comment:
there is a 'p' missing in the file name : "CarConsum**p**tion.test.js"

Reply:
Yeah you're right. I'll correct that